dbus-opendtu.py

```diff
@@ -441,22 +482,32 @@ def main():
             '/Ac/L2/Energy/Forward': {'initial': None, 'textformat': _kwh},
             '/Ac/L3/Energy/Forward': {'initial': None, 'textformat': _kwh},
           }
+      
+      if dtu != 'template':
+        service = DbusService(
+            servicename='com.victronenergy.pvinverter',
+            paths=paths,
+            actual_inverter=0) 
 
-      service = DbusService(
-          servicename='com.victronenergy.pvinverter',
-          paths=paths,
-          actual_inverter=0) 
+        number_of_inverters = service._getNumberOfInverters()
 
-      number_of_inverters = service._getNumberOfInverters()
+        if number_of_inverters > 1:
+          #start our main-service if there are more than 1 inverter
+          for actual_inverter in range(number_of_inverters -1):
+            DbusService(
+              servicename='com.victronenergy.pvinverter',
+              paths=paths,
+              actual_inverter=actual_inverter+1)
 
-      if number_of_inverters > 1:
-        #start our main-service if there are more than 1 inverter
-        for actual_inverter in range(number_of_inverters -1):
-          DbusService(
+      for actual_template in range(number_of_templates):
```

Comment:
Ah - jetzt kommt's mir wieder: Es kann immer genau einen DTU geben (Ahoy oder OpenDTU) - ggf. mit mehr als 1 Inverter und außerdem 0 bis n Templates. Das sollten wir in Readme und Code deutlich reinschreiben. Das muss man vor Augen haben - sowohl beim Benutzen als auch beim Lesen des Code.

Reply:
Ja hast du recht, die Doku Seite muss ich auch noch anpassen danach

Reply:
Nicht vergessen....

Reply:
done https://github.com/henne49/dbus-opendtu/blob/Mutli-Template/README.md